### ISSUE TITLE
Problem synchronisation of new products and image in offline mode

### ISSUE THREAD

**Issue by JeremyGfeller** [DivanteLtd/vue-storefront#3365]
I have a problem, when I add a new product in Magento, the product doesn't come in vue-storefront, but if I update the price (in Magento), he is updated in vue-storefront. 

The second problem is for the images, when I put the navigator in offline mode, all the pictures disapear and become all white. 

I hope anyone can help me

**Comment by JeremyGfeller** [comment]
* someone, sorry

**Comment by pkarw** [comment]
@JeremyGfeller thanks for contacting us. are you using `mage2vuestorefront` indexer? If so please check if you're properly using the the [delta sync mode](https://github.com/DivanteLtd/mage2vuestorefront#delta-indexer). Please check the https://github.com/DivanteLtd/magento2-vsbridge-indexer as well

@alinadivante can you please check the offline mode pictures? Is it on: `demo`, `next` or `test.storefrontcloud.io` instances?

**Comment by JeremyGfeller** [comment]
Thanks for your answer, yes I use `mage2vuestorefront` I launched this command in the directory of `mage2vuestorefront` : 
> node --harmony cli.js productsdelta --partitions=1

It means, I have to run every time this command when I create, update, or delete a product or a price or anything else for a product ? 
Or there is another way to synchronize everything automatically ?

**Comment by pkarw** [comment]
Correct, it should be scheduled in crontab; the other way is to use native indexer (magento2-vsbridge-indexer project)

**Comment by JeremyGfeller** [comment]
With the native indexer, it synchronize everything automatically ?

**Comment by pkarw** [comment]
that’s  correct

**Comment by JeremyGfeller** [comment]
Does it normal, I have to put credential when I want to download to Magento folder ? 
![Capture d’écran 2019-08-15 à 11 23 50](https://user-images.githubusercontent.com/36196039/63086004-31c7a580-bf4f-11e9-9e76-ad00079b7164.png)

**Comment by JeremyGfeller** [comment]
So I installed your magento2-vsbridge-indexer project but there is nothing update automatically. Do you need something from my configuration and the command i've made in the terminal ?

**Comment by alinadivante** [comment]
I have the same problem I wrote about here https://github.com/DivanteLtd/vue-storefront/issues/3263#issuecomment-519890661
I see new product added only on Magento front, not in VSF.
@pkarw Maybe something is wrong with fullreindex.sh script or maybe I'm doing something wrong. I need clear instructions for re-index and re-import processes.

**Comment by pkarw** [comment]
@alinadivante please contact me today regarding this one

**Comment by JeremyGfeller** [comment]
I always have the problem guys. I installed magento2-vsbridge-indexer but nothing synchronize automatically
